subworkflows/local/variantcalling_csv.nf

```diff
@@ -13,6 +13,6 @@ workflow VARIANTCALLING_CSV {
             sample        = meta.id
             variantcaller = meta.variantcaller
             vcf = "${params.outdir}/variant_calling/${meta.id}/${variantcaller}/${vcf.getName()}"
-            ["variantcalled.csv", "patient,gender,sample,variantcaller,vcf\n${patient},${sample},${variantcaller},${vcf}\n"]
+            ["variantcalled.csv", "patient,sample,variantcaller,vcf\n${patient},${sample},${variantcaller},${vcf}\n"]
```

Comment:
not sure if we want to remove the sex field here or not, but figured that this was a typo anyway

Reply:
@FriederikeHanssen what about that one?